docs/HOL_Continuous-Deployment-Using-Custom-Agent/README.md

```diff
@@ -0,0 +1,121 @@
+
+    ![](<media/create_agent_pool.png>) 
+
+3. Instead of manually creating the VM in Azure, we are going to use an Azure Resource Management (ARM) template. Simply click the **Deploy to Azure** button below and follow the wizard to deploy the machine. You will need to log in to the Azure Portal.
+                                                                    
+    <a href="https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2FMicrosoft%2FPartsUnlimitedMRP%2Fmaster%2Fdocs%2FHOL_Continuous-Deployment-Using-Custom-Agent%2Fenv%2FContinuousDeploymentCustomAgentPartsUnlimitedMRP.json" target="_blank">
+        <img src="http://azuredeploy.net/deploybutton.png"/>
+    </a>
+    <a href="http://armviz.io/#/?load=https%3A%2F%2Fraw.githubusercontent.com%2FMicrosoft%2FPartsUnlimitedMRP%2Fmaster%2Fdocs%2FHOL_Continuous-Deployment-Using-Custom-Agent%2Fenv%2FContinuousDeploymentCustomAgentPartsUnlimitedMRP.json" target="_blank">
+        <img src="http://armviz.io/visualizebutton.png"/>
+    </a>
+
+    The VMs will be deployed to a Resource Group along with a virtual network (VNET) and some other required resources. You can 
+    delete the resource group in order to remove all the created resources at any time.
+
+4. You will need to select a subscription and region to deploy the Resource Group to and supply an admin username, password, and unique name for the machine. The machine will be a Standard A2.
+
+    ![](<media/set_arm_parameters.png>)
+
+    Make sure you make a note of the region as well as the username and password for the machine. Allow about 10 minutes for deployment and then another 10 minutes for the VSTS agent and MRP dependency configuration. 
+
+5. You will also need to specify the VSTS account to use (the DNS name before *visualstudio.com*) and a personal access token. If you don't have a personal access token, follow [this link](https://www.visualstudio.com/en-us/docs/setup-admin/team-services/use-personal-access-tokens-to-authenticate) to create one.
+
+6. When the deployment completes, you should see the following resources in the Azure Portal:
+
+    ![](<media/post_deployment_rg.png>)
+
+    Click on the "partsmrp" Public IP Address. Then make a note of the DNS name:
+
+    ![](<media/public_ip_dns.png>)
+
+    >**Note:** The lab requires several ports to be open, such as SSH ports and the Parts Unlimited MRP app port on the partsmrp machine. 
+	The ARM template opens these ports on the machine for you.
+
+### Task 2: Create release definition ###
+
+1. At the homepage of the PartsUnlimitedMRP team project in Visual Studio Team Services, click on the **Release** tab in the upper-left corner of the page. Then, click the **+** button on the left and choose **Create new release definition**.
+
+    ![](<media/create_release_definition.png>)
```

Comment:
If the project doesn't have a definition yet, the screenshot should be : 
![new release def](https://cloud.githubusercontent.com/assets/8661347/17606970/88fe7ec6-5ff0-11e6-93e5-0c115d87c26b.png)